### ISSUE TITLE
User Review

### ISSUE THREAD

**Issue by edavidj** [OpenLiberty/guide-mongodb-intro#37]
- [x] Record the time it takes to go through the entire guide
- [x] Test each step in the guide is clear and easy to understand
- [x] Ensure no logic errors
- [x] Resolve anything that does not work
- [x] Resolve anything usability issues (confusing or hard to follow)
- [x] Achieve basic understanding/working knowledge of the topic in the guide

Comments:
- Time: 25 min
- Not really something to fix, more to note, but the sections on the mongodb setup were dense to read with the large amount of hotlinks and inline blocks in them. I understand that they're helpful in connecting what you're reading to the corresponding section in the Dockerfile, but as a user I would probably skip over that block entirely.
- I think it would be nice to give them a quick command to copy at the end to remove the mongodb container running, or remind them to do so themselves.
- No logical or technical errors encountered.
- Achieved solid understanding of the content.

**Comment by jimmy1wu** [comment]
@edavidj hello david, thank you for the review. i have shortened the setup section and added a clean up section at the end. the changes are available on the lgdev1 site. feel free to add any additional comments here or in https://github.com/OpenLiberty/draft-guide-mongodb/pull/39.

**Comment by jimmy1wu** [comment]
@edavidj approves, closing the issue
